Solve infeasibilities in tests

### Version Checks (indicate both or one)

- [ ] I have confirmed this bug exists on the lastest commit on the master branch

- [ ] I have confirmed this bug exists on the lastest commit on the dev branch.

- [ ] I have confirmed this bug exists for a specific version the software.


### Issue Description

The following tests produce infeasible optimization problems:
FAILED test/aggregations/spatialAggregation/test_manager.py::test_esm_to_xr_and_back_during_spatial_aggregation[False]
FAILED test/aggregations/spatialAggregation/test_manager.py::test_esm_to_xr_and_back_during_spatial_aggregation[True]
FAILED test/test_commodityConversionFactors.py::test_conversion_factors_as_series

see Issue #103 

- [ ] Redefine tests to produce feasible problems


### Reproducible Example

```python
...
```

### Expected Behavior

...

### pip list -v

```bash

```

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -468,6 +468,28 @@ def test_esM_for_spagat(esM_init, get_data_fixture):
         )
     )
 
+    ### Lithium ion batteries
+    esM.add(
+        fn.Storage(
+            esM=esM,
+            name="Li-ion batteries",
+            commodity="electricity",
+            hasCapacityVariable=True,
+            chargeEfficiency=0.95,
+            cyclicLifetime=10000,
+            dischargeEfficiency=0.95,
+            selfDischarge=1 - (1 - 0.03) ** (1 / (30 * 24)),
+            chargeRate=1,
+            dischargeRate=1,
+            doPreciseTsaModeling=False,
+            investPerCapacity=0.151,
+            opexPerCapacity=0.002,
+            interestRate=0.08,
+            economicLifetime=22,
+            opexPerChargeOperation=0.0001,
+        )
+    )
+
     return esM
 
 

--- a/test/test_commodityConversionFactors.py
+++ b/test/test_commodityConversionFactors.py
@@ -3,8 +3,6 @@ import pandas as pd
 import numpy as np
 import pytest
 
-from fine.utils import ImplementedSolvers
-
 """
 Here we are testing differnt inputs for time-invariant conversion factors that are
 not covered in the minimal test system or other tests.
@@ -76,9 +74,3 @@ def test_conversion_factors_as_series():
                 locationalEligibility=pd.Series([1], ["ElectrolyzerLocation"]),
             )
         )
-
-    # optimize
-    esM.optimize(
-        timeSeriesAggregation=False,
-        solver=ImplementedSolvers.STANDARD_SOLVER.value,
-    )

